Casks/font-firacode-nerd-font-mono.rb

```diff
@@ -1,15 +1,15 @@
 cask 'font-firacode-nerd-font-mono' do
-  version '2.0.0'
-  sha256 '09894d24bf3d61493dba052187a9200497135a4b885cb837bcb637ad2e62070f'
+  version '2.1.0'
+  sha256 '21de9aa0edaa3fd2dc1d87fb9ecec0b67c9b3b18bd1998a19904158067fea7e7'
 
   url "https://github.com/ryanoasis/nerd-fonts/releases/download/v#{version}/FiraCode.zip"
   appcast 'https://github.com/ryanoasis/nerd-fonts/releases.atom'
-  name 'FuraCode Nerd Font (FiraCode)'
-  homepage 'https://github.com/ryanoasis/nerd-fonts'
+  name 'FiraCode Nerd Font (FiraCode)'
```

Comment:
```suggestion
  name 'FiraCode Nerd Font'
```

Since the name is being changes from `FuraCode` to `FiraCode` this is redundant.

Reply:
@parkerbxyz I just followed what seems [like](https://github.com/Homebrew/homebrew-cask-fonts/blob/master/Casks/font-dejavusansmono-nerd-font.rb) [the](https://github.com/Homebrew/homebrew-cask-fonts/blob/master/Casks/font-firacode-nerd-font.rb) [current](https://github.com/Homebrew/homebrew-cask-fonts/blob/master/Casks/font-hack-nerd-font-mono.rb) [pratice](https://github.com/Homebrew/homebrew-cask-fonts/blob/master/Casks/font-inconsolata-nerd-font.rb).
Please confirm if you'd like me to go ahead anyway.

Reply:
Oh! Interesting. I'd leave it up to the maintainers of this repository in that case. Thanks, @julienma!

Reply:
> I'd leave it up to the maintainers of this repository in that case.

[Cue Futurama](https://youtu.be/LdSzXpmY2ic).